cranelift/filetests/filetests/runtests/simd-regtype.clif

```diff
@@ -0,0 +1,32 @@
+test run
```

Comment:
I don't think this actually runs anything without `; run(...)` comments, right?

Reply:
Ah, yes, good point. I played with this for a bit but ran into other issues (with the vmctx) and concluded it was simpler just to add the Wasm test directly, rather than capture its CLIF.